python/test/extrinsics/test_universal_microdata_scheme.py

```diff
@@ -0,0 +1,197 @@
+import itertools as it
+import functools as ft
+
+import pytest
+
+import opendp.prelude as dp
+from opendp.mod import Domain, Metric, Transformation
+from opendp._extrinsics._utilities import with_privacy
+
+from ..helpers import optional_dependency
+
+dp.enable_features("contrib", "floating-point")
+
+
+def test_microdataa_faithfulness():
+
+    from opendp._extrinsics.microdata.universal_microdata_scheme import make_private_universal_microdata_scheme
+
+    np = pytest.importorskip("numpy")
+    pl = pytest.importorskip("polars")
+
+    COLS = list("ABC")
+    NUM_RECORDS = 5
+
+    real_df = pl.LazyFrame({
+        "A": [0, 1, 0, 1, 0],
+        "B": [1, 0, 1, 0, 1],
+        "C": [0, 0, 1, 1, 1]
+    })
+
+    lf_domain = dp.lazyframe_domain(
+        [dp.series_domain(col, dp.atom_domain(T=dp.i32))
+         for col in COLS])
+
+    lf_domain_with_margin = dp.with_margin(lf_domain,
+                                           by=COLS, public_info="keys")
+
+    def similarity_record_fn(x, y):
+        return np.sum(np.abs((x - y)))
+
+    with (optional_dependency("numpy"),
+          optional_dependency("polars"),
```

Comment:
```suggestion
```

Are all of these necessary? I just haven't seen where the code was using polars, but maybe it's indirect?